ragstack/colbert/colbert_embedding.py

```diff
@@ -123,17 +123,15 @@ def __init__(
             self.checkpoint = self.checkpoint.cuda()
 
     def embed_documents(
```

Comment:
is "document" correct here? 
if so, `embed_query` should be embed_document

Probably we should use embed_texts and embed_text

Reply:
this is just for implementing the abstract method in this class: https://github.com/datastax/ragstack-ai/pull/347/files#diff-c553042020c3e53b973244f9a67fb153ece342591f1ed32b563f97ce7f2ff0faR76

Which I'm not sure if we need. What do you think?

Reply:
yes let's keep the interface, code is much leaner to read. Wonder if we want to rename the interface methods too